src/Features/CSharp/Portable/ReplaceMethodWithProperty/CSharpReplaceMethodWithPropertyService.cs

```diff
@@ -396,7 +396,7 @@ public void ReplaceSetReference(SyntaxEditor editor, SyntaxToken nameToken, stri
             }
 
             // It was invoked.  Remove the invocation, and also change the name if necessary.
-            replace(editor, invocation, nameNode, newName);
+            replace(editor, invocation, nameNode, newName.WithTriviaFrom(invocation));
```

Comment:
this looks like this will blow away the trivia decisions we made here: 
https://github.com/dotnet/roslyn/pull/61150/files#diff-6a9d63bb896c84bd0ea28a981c920e2fae3a671fefd9ed7eccb1dec036b3bb3bL378
and here:
https://github.com/dotnet/roslyn/pull/61150/files#diff-6a9d63bb896c84bd0ea28a981c920e2fae3a671fefd9ed7eccb1dec036b3bb3bR385

Can we reconcile this?

Reply:
weird!  basically, go up in this file to `if (nameChanged)` block.  THere we are already doing some amount of trivia handling.  It looks like this will override that work.

Reply:
This was handling trivia only for cases, when the name is changed and for some reason did it in very overcomplicated way). Simplified and added tests when no invocation syntax is present